tag_release

```diff
@@ -4,6 +4,22 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+git_confirm() {
+    local response
+    if [[ ${FLAGS_force} -ne ${FLAGS_TRUE} ]]; then
+        echo "Executing: git $@"
```

Comment:
Is there a reason to use echo instead of info?

Reply:
It's not really a log message (e.g. it isn't shown if `--force` is given); it's part of the prompt.